src/components/Header.js

```diff
@@ -1,10 +1,61 @@
-import React from 'react';
-import styled from 'styled-components';
+import React from "react";
+import styled from "styled-components";
+import { images } from "./images/image";
 
-export default function Header() {
-	return <Wrapper>발신자 변경해주는 상단바</Wrapper>;
+export default function Header(props) {
```

Comment:
props를 비구조화할당 문법으로 받아옵니당

```suggestion
export default function Header({ user, onClick, imgUrl }) {
```

Reply:
안그래도 "props.~~~" 처럼 props를 붙여주는게 지저분해 보였는데 비구조할당을 쓰면 되는군요! 감사합니다